Cargo.lock

```diff
@@ -402,11 +490,11 @@ checksum = "619743e34b5ba4e9703bba34deac3427c72507c7159f5fd030aea8cac0cfe341"
 
 [[package]]
 name = "async-channel"
-version = "1.7.1"
+version = "1.8.0"
```

Comment:
Not sure how async-channel doesn't end up with 2 different versions

https://discord.com/channels/691052431525675048/1082044238189043712/1082044238189043712